mava/components/jax/building/parameter_client.py

```diff
@@ -194,3 +194,171 @@ def on_building_trainer_parameter_client(self, builder: SystemBuilder) -> None:
     def name() -> str:
         """Component type name, e.g. 'dataset' or 'executor'."""
         return "trainer_parameter_client"
+
+
+####################
+# SEPARATE NETWORKS
+###################
+
+
+class ExecutorParameterClientSeparateNetworks(BaseParameterClient):
+    def __init__(
+        self,
+        config: ExecutorParameterClientConfig = ExecutorParameterClientConfig(),
+    ) -> None:
+        """Parameter client
+
+        Args:
+            config : parameter client config
+        """
+
+        self.config = config
+
+    def on_building_executor_parameter_client(self, builder: SystemBuilder) -> None:
+        """_summary_
+
+        Args:
+            builder : _description_
+        """
+        # Create policy parameters
+        params = {}
+        get_keys = []
+        net_type_key = "networks"
+        for agent_net_key in builder.store.networks[net_type_key].keys():
```

Comment:
E.g. could we not just specify the networks in a specific way (in the networks components) so that this can be one for loop over all networks?

Reply:
I think we might even be able to already have this done in one loop as is.

Reply:
Indeed, it is now in only one loop.

Reply:
Great thanks!